packages/ui/src/models/visualization/flows/camel-rest-visual-entity.ts

```diff
@@ -13,17 +13,26 @@ import {
   NodeInteraction,
   VisualComponentSchema,
 } from '../base-visual-entity';
+import { AbstractCamelVisualEntity } from './abstract-camel-visual-entity';
 import { CamelCatalogService } from './camel-catalog.service';
 import { NodeMapperService } from './nodes/node-mapper.service';
-import { AbstractCamelVisualEntity } from './abstract-camel-visual-entity';
 
 export class CamelRestVisualEntity extends AbstractCamelVisualEntity<{ rest: Rest }> implements BaseVisualCamelEntity {
   id: string;
   readonly type = EntityType.Rest;
   private schemaValidator: ValidateFunction<Rest> | undefined;
-
-  constructor(public rest: { rest: Rest }) {
-    super(rest);
+  private readonly OMIT_FORM_FIELDS = [
+    ...SchemaService.OMIT_FORM_FIELDS,
+    'get',
+    'post',
+    'put',
+    'delete',
+    'head',
+    'patch',

```

Comment:
Here we're removing these properties from the form, since it would be better to configure them through the UI, like adding nodes and stuff.